BUG/TST: stats: XSLOW test failures of `ttest_ind`

I get some new errors when I run the XSLOW tests.  Here are a couple from `stats`, where it looks like the problem is in the test: the parameter `permutations` is passed to [`ttest_ind`](https://scipy.github.io/devdocs/reference/generated/scipy.stats.ttest_ind.html), but that function doesn't accept that parameter.

<details>
<summary>Test run</summary>

```
$ SCIPY_XSLOW=1 spin test -m full -j 12 -t scipy/stats/tests/test_stats.py
Invoking `build` prior to running tests:
$ meson compile -j 12 -C build
INFO: autodetecting backend as ninja
INFO: calculating backend command to run: /home/warren/repos/git/forks/scipy-fresh/venv/bin/ninja -C /home/warren/repos/git/forks/scipy-fresh/build -j 12
ninja: Entering directory `/home/warren/repos/git/forks/scipy-fresh/build'
[2/338] Generating subprojects/highs/src/HConfig.h with a custom command
$ meson install --only-changed -C build --destdir ../build-install --tags=runtime,python-runtime,tests,devel
$ export PYTHONPATH="/home/warren/repos/git/forks/scipy-fresh/build-install/usr/lib/python3.13/site-packages"
$ /home/warren/repos/git/forks/scipy-fresh/venv/bin/python3 -P -c 'import scipy'
$ cd /home/warren/repos/git/forks/scipy-fresh/build-install/usr/lib/python3.13/site-packages
$ /home/warren/repos/git/forks/scipy-fresh/venv/bin/python3 -P -m pytest -n 12 scipy/stats/tests/test_stats.py
============================= test session starts ==============================
platform linux -- Python 3.13.3, pytest-8.4.1, pluggy-1.6.0
rootdir: /home/warren/repos/git/forks/scipy-fresh
configfile: pytest.ini
plugins: anyio-4.9.0, timeout-2.4.0, hypothesis-6.135.14, xdist-3.7.0, cov-6.2.1
12 workers [1198 items]   
........................................................................ [  6%]
........................................................................ [ 12%]
........................................................................ [ 18%]
........................................................................ [ 24%]
........................................................................ [ 30%]
........................................................................ [ 36%]
........................................................................ [ 42%]
.............ss......................................................... [ 48%]
........................................................................ [ 54%]
........................................................................ [ 60%]
....x.........x......................................................... [ 66%]
..........................................x............................. [ 72%]
..............................................................F......... [ 78%]
........................................................................ [ 84%]
........................................................................ [ 90%]
.....................s...................................F.............. [ 96%]
..............................................                           [100%]
=================================== FAILURES ===================================
______ Test_ttest_ind_common.test_ttest_many_dims[equal_var-permutations] ______
[gw8] linux -- Python 3.13.3 /home/warren/repos/git/forks/scipy-fresh/venv/bin/python3

self = <scipy.stats.tests.test_stats.Test_ttest_ind_common object at 0x7638c87165d0>
kwds = {'permutations': 200, 'random_state': 0}, equal_var = True

    @pytest.mark.xslow()
    @pytest.mark.parametrize("kwds", [{'permutations': 200, 'random_state': 0},
                                      {'trim': .2}, {}],
                             ids=["permutations", "trim", "basic"])
    @pytest.mark.parametrize('equal_var', [True, False],
                             ids=['equal_var', 'unequal_var'])
    def test_ttest_many_dims(self, kwds, equal_var):
        # Test that test works on many-dimensional arrays
        np.random.seed(0)
        a = np.random.rand(5, 4, 4, 7, 1, 6)
        b = np.random.rand(4, 1, 8, 2, 6)
>       res = stats.ttest_ind(a, b, axis=-3, **kwds)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

a          = array([[[[[[5.48813504e-01, 7.15189366e-01, 6.02763376e-01,
            5.44883183e-01, 4.23654799e-01, 6.45894113e-01...e-01, 7.31450249e-01,
            2.68878230e-01, 3.05965591e-01, 3.05472980e-01]]]]]],
      shape=(5, 4, 4, 7, 1, 6))
b          = array([[[[[0.87260193, 0.22609761, 0.56240183, 0.99993115, 0.72846549,
           0.33635066],
          [0.76545897, ...        0.79329065],
          [0.33568778, 0.67971569, 0.03267317, 0.39378824, 0.0196105 ,
           0.81634554]]]]])
equal_var  = True
kwds       = {'permutations': 200, 'random_state': 0}
self       = <scipy.stats.tests.test_stats.Test_ttest_ind_common object at 0x7638c87165d0>

scipy/stats/tests/test_stats.py:5437: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

_no_deco = False, kwds = {'permutations': 200, 'random_state': 0}
temp = array([[[[[[5.48813504e-01, 7.15189366e-01, 6.02763376e-01,
            5.44883183e-01, 4.23654799e-01, 6.45894113e-01...e-01, 7.31450249e-01,
            2.68878230e-01, 3.05965591e-01, 3.05472980e-01]]]]]],
      shape=(5, 4, 4, 7, 1, 6))
params = ['a', 'b', 'axis', 'equal_var', 'nan_policy', 'alternative', ...]
maxarg = 2
d_args = {'a': array([[[[[[5.48813504e-01, 7.15189366e-01, 6.02763376e-01,
            5.44883183e-01, 4.23654799e-01, 6.458941...       0.79329065],
          [0.33568778, 0.67971569, 0.03267317, 0.39378824, 0.0196105 ,
           0.81634554]]]]])}
intersection = set()

    @wraps(hypotest_fun_in)
    def axis_nan_policy_wrapper(*args, _no_deco=False, **kwds):
    
        if _no_deco:  # for testing, decorator does nothing
            return hypotest_fun_in(*args, **kwds)
    
        # For now, skip the decorator entirely if using array API. In the future,
        # we'll probably want to use it for `keepdims`, `axis` tuples, etc.
        if len(args) == 0:  # extract sample from `kwds` if there are no `args`
            used_kwd_samples = list(set(kwds).intersection(set(kwd_samples)))
            temp = used_kwd_samples[:1]
        else:
            temp = args[0]
    
        if not is_numpy(array_namespace(temp)):
            msg = ("Use of `nan_policy` and `keepdims` "
                   "is incompatible with non-NumPy arrays.")
            if 'nan_policy' in kwds or 'keepdims' in kwds:
                raise NotImplementedError(msg)
            return hypotest_fun_in(*args, **kwds)
    
        # We need to be flexible about whether position or keyword
        # arguments are used, but we need to make sure users don't pass
        # both for the same parameter. To complicate matters, some
        # functions accept samples with *args, and some functions already
        # accept `axis` and `nan_policy` as positional arguments.
        # The strategy is to make sure that there is no duplication
        # between `args` and `kwds`, combine the two into `kwds`, then
        # the samples, `nan_policy`, and `axis` from `kwds`, as they are
        # dealt with separately.
    
        # Check for intersection between positional and keyword args
        params = list(inspect.signature(hypotest_fun_in).parameters)
        if n_samples is None:
            # Give unique names to each positional sample argument
            # Note that *args can't be provided as a keyword argument
            params = [f"arg{i}" for i in range(len(args))] + params[1:]
    
        # raise if there are too many positional args
        maxarg = (np.inf if inspect.getfullargspec(hypotest_fun_in).varargs
                  else len(inspect.getfullargspec(hypotest_fun_in).args))
        if len(args) > maxarg:  # let the function raise the right error
            hypotest_fun_in(*args, **kwds)
    
        # raise if multiple values passed for same parameter
        d_args = dict(zip(params, args))
        intersection = set(d_args) & set(kwds)
        if intersection:  # let the function raise the right error
            hypotest_fun_in(*args, **kwds)
    
        # Consolidate other positional and keyword args into `kwds`
        kwds.update(d_args)
    
        # rename avoids UnboundLocalError
        if callable(n_samples):
            # Future refactoring idea: no need for callable n_samples.
            # Just replace `n_samples` and `kwd_samples` with a single
            # list of the names of all samples, and treat all of them
            # as `kwd_samples` are treated below.
            n_samp = n_samples(kwds)
        else:
            n_samp = n_samples or len(args)
    
        # get the number of outputs
        n_out = n_outputs  # rename to avoid UnboundLocalError
        if callable(n_out):
            n_out = n_out(kwds)
    
        # If necessary, rearrange function signature: accept other samples
        # as positional args right after the first n_samp args
        kwd_samp = [name for name in kwd_samples
                    if kwds.get(name, None) is not None]
        n_kwd_samp = len(kwd_samp)
        if not kwd_samp:
            hypotest_fun_out = hypotest_fun_in
        else:
            def hypotest_fun_out(*samples, **kwds):
                new_kwds = dict(zip(kwd_samp, samples[n_samp:]))
                kwds.update(new_kwds)
                return hypotest_fun_in(*samples[:n_samp], **kwds)
    
        # Extract the things we need here
        try:  # if something is missing
            samples = [np.atleast_1d(kwds.pop(param))
                       for param in (params[:n_samp] + kwd_samp)]
        except KeyError:  # let the function raise the right error
            # might need to revisit this if required arg is not a "sample"
            hypotest_fun_in(*args, **kwds)
        vectorized = True if 'axis' in params else False
        vectorized = vectorized and not override['vectorization']
        axis = kwds.pop('axis', default_axis)
        nan_policy = kwds.pop('nan_policy', 'propagate')
        keepdims = kwds.pop("keepdims", False)
        del args  # avoid the possibility of passing both `args` and `kwds`
    
        # convert masked arrays to regular arrays with sentinel values
        samples, sentinel = _masked_arrays_2_sentinel_arrays(samples)
    
        # standardize to always work along last axis
        reduced_axes = axis
        if axis is None:
            if samples:
                # when axis=None, take the maximum of all dimensions since
                # all the dimensions are reduced.
                n_dims = np.max([sample.ndim for sample in samples])
                reduced_axes = tuple(range(n_dims))
            samples = [np.asarray(sample.ravel()) for sample in samples]
        else:
            # don't ignore any axes when broadcasting if paired
            samples = _broadcast_arrays(samples, axis=axis if not paired else None)
            axis = np.atleast_1d(axis)
            n_axes = len(axis)
            # move all axes in `axis` to the end to be raveled
            samples = [np.moveaxis(sample, axis, range(-len(axis), 0))
                       for sample in samples]
            shapes = [sample.shape for sample in samples]
            # New shape is unchanged for all axes _not_ in `axis`
            # At the end, we append the product of the shapes of the axes
            # in `axis`. Appending -1 doesn't work for zero-size arrays!
            new_shapes = [shape[:-n_axes] + (np.prod(shape[-n_axes:]),)
                          for shape in shapes]
            samples = [sample.reshape(new_shape)
                       for sample, new_shape in zip(samples, new_shapes)]
        axis = -1  # work over the last axis
        NaN = _get_nan(*samples) if samples else np.nan
    
        # if axis is not needed, just handle nan_policy and return
        ndims = np.array([sample.ndim for sample in samples])
        if np.all(ndims <= 1):
            # Addresses nan_policy == "raise"
            if nan_policy != 'propagate' or override['nan_propagation']:
                contains_nan = [_contains_nan(sample, nan_policy)
                                for sample in samples]
            else:
                # Behave as though there are no NaNs (even if there are)
                contains_nan = [False] * len(samples)
    
            # Addresses nan_policy == "propagate"
            if any(contains_nan) and (nan_policy == 'propagate'
                                      and override['nan_propagation']):
                res = np.full(n_out, NaN)
                res = _add_reduced_axes(res, reduced_axes, keepdims)
                return tuple_to_result(*res)
    
            # Addresses nan_policy == "omit"
            too_small_msg = too_small_1d_not_omit
            if any(contains_nan) and nan_policy == 'omit':
                # consider passing in contains_nan
                samples = _remove_nans(samples, paired)
                too_small_msg = too_small_1d_omit
    
            if sentinel:
                samples = _remove_sentinel(samples, paired, sentinel)
    
            if is_too_small(samples, kwds):
                warnings.warn(too_small_msg, SmallSampleWarning, stacklevel=2)
                res = np.full(n_out, NaN)
                res = _add_reduced_axes(res, reduced_axes, keepdims)
                return tuple_to_result(*res)
    
            res = hypotest_fun_out(*samples, **kwds)
            res = result_to_tuple(res, n_out)
            res = _add_reduced_axes(res, reduced_axes, keepdims)
            return tuple_to_result(*res)
    
        # check for empty input
        empty_output = _check_empty_inputs(samples, axis)
        # only return empty output if zero sized input is too small.
        if (
            empty_output is not None
            and (is_too_small(samples, kwds) or empty_output.size == 0)
        ):
            if is_too_small(samples, kwds) and empty_output.size != 0:
                warnings.warn(too_small_nd_not_omit, SmallSampleWarning,
                              stacklevel=2)
            res = [empty_output.copy() for i in range(n_out)]
            res = _add_reduced_axes(res, reduced_axes, keepdims)
            return tuple_to_result(*res)
    
        # otherwise, concatenate all samples along axis, remembering where
        # each separate sample begins
        lengths = np.array([sample.shape[axis] for sample in samples])
        split_indices = np.cumsum(lengths)
        x = _broadcast_concatenate(samples, axis, paired=paired)
    
        # Addresses nan_policy == "raise"
        if nan_policy != 'propagate' or override['nan_propagation']:
            contains_nan = _contains_nan(x, nan_policy)
        else:
            contains_nan = False  # behave like there are no NaNs
    
        if vectorized and not contains_nan and not sentinel:
>           res = hypotest_fun_out(*samples, axis=axis, **kwds)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E           TypeError: ttest_ind() got an unexpected keyword argument 'permutations'

NaN        = np.float64(nan)
_no_deco   = False
axis       = -1
contains_nan = np.False_
d_args     = {'a': array([[[[[[5.48813504e-01, 7.15189366e-01, 6.02763376e-01,
            5.44883183e-01, 4.23654799e-01, 6.458941...       0.79329065],
          [0.33568778, 0.67971569, 0.03267317, 0.39378824, 0.0196105 ,
           0.81634554]]]]])}
default_axis = 0
empty_output = None
hypotest_fun_in = <function ttest_ind at 0x7638fc336de0>
hypotest_fun_out = <function ttest_ind at 0x7638fc336de0>
intersection = set()
is_too_small = <function _axis_nan_policy_factory.<locals>.is_too_small at 0x7638fc336ca0>
keepdims   = False
kwd_samp   = []
kwd_samples = ()
kwds       = {'permutations': 200, 'random_state': 0}
lengths    = array([7, 8])
maxarg     = 2
n_axes     = 1
n_kwd_samp = 0
n_out      = 6
n_outputs  = 6
n_samp     = 2
n_samples  = 2
nan_policy = 'propagate'
ndims      = array([6, 6])
new_shapes = [(5, 4, 4, 2, 6, np.int64(7)), (5, 4, 4, 2, 6, np.int64(8))]
override   = {'nan_propagation': True, 'vectorization': False}
paired     = False
params     = ['a', 'b', 'axis', 'equal_var', 'nan_policy', 'alternative', ...]
reduced_axes = -3
result_to_tuple = <function unpack_TtestResult at 0x7638fc3362a0>
samples    = [array([[[[[[5.48813504e-01, 4.37587211e-01, 5.68044561e-01, ...,
            1.18274426e-01, 2.64555612e-01, 6.120957...40454229, 0.97580369, 0.08510591, ..., 0.39058309,
            0.05805632, 0.81634554]]]]]], shape=(5, 4, 4, 2, 6, 8))]
sentinel   = None
shapes     = [(5, 4, 4, 2, 6, 7), (5, 4, 4, 2, 6, 8)]
split_indices = array([ 7, 15])
temp       = array([[[[[[5.48813504e-01, 7.15189366e-01, 6.02763376e-01,
            5.44883183e-01, 4.23654799e-01, 6.45894113e-01...e-01, 7.31450249e-01,
            2.68878230e-01, 3.05965591e-01, 3.05472980e-01]]]]]],
      shape=(5, 4, 4, 7, 1, 6))
tuple_to_result = <function pack_TtestResult at 0x7638fc336020>
vectorized = True
x          = array([[[[[[0.5488135 , 0.43758721, 0.56804456, ..., 0.57115297,
            0.68493794, 0.30073256],
           [0.71...63567726, 0.49569992, 0.62663675, ..., 0.39058309,
            0.05805632, 0.81634554]]]]]], shape=(5, 4, 4, 2, 6, 15))

scipy/stats/_axis_nan_policy.py:611: TypeError
_____ Test_ttest_ind_common.test_ttest_many_dims[unequal_var-permutations] _____
[gw8] linux -- Python 3.13.3 /home/warren/repos/git/forks/scipy-fresh/venv/bin/python3

self = <scipy.stats.tests.test_stats.Test_ttest_ind_common object at 0x7638d119ed70>
kwds = {'permutations': 200, 'random_state': 0}, equal_var = False

    @pytest.mark.xslow()
    @pytest.mark.parametrize("kwds", [{'permutations': 200, 'random_state': 0},
                                      {'trim': .2}, {}],
                             ids=["permutations", "trim", "basic"])
    @pytest.mark.parametrize('equal_var', [True, False],
                             ids=['equal_var', 'unequal_var'])
    def test_ttest_many_dims(self, kwds, equal_var):
        # Test that test works on many-dimensional arrays
        np.random.seed(0)
        a = np.random.rand(5, 4, 4, 7, 1, 6)
        b = np.random.rand(4, 1, 8, 2, 6)
>       res = stats.ttest_ind(a, b, axis=-3, **kwds)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

a          = array([[[[[[5.48813504e-01, 7.15189366e-01, 6.02763376e-01,
            5.44883183e-01, 4.23654799e-01, 6.45894113e-01...e-01, 7.31450249e-01,
            2.68878230e-01, 3.05965591e-01, 3.05472980e-01]]]]]],
      shape=(5, 4, 4, 7, 1, 6))
b          = array([[[[[0.87260193, 0.22609761, 0.56240183, 0.99993115, 0.72846549,
           0.33635066],
          [0.76545897, ...        0.79329065],
          [0.33568778, 0.67971569, 0.03267317, 0.39378824, 0.0196105 ,
           0.81634554]]]]])
equal_var  = False
kwds       = {'permutations': 200, 'random_state': 0}
self       = <scipy.stats.tests.test_stats.Test_ttest_ind_common object at 0x7638d119ed70>

scipy/stats/tests/test_stats.py:5437: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

_no_deco = False, kwds = {'permutations': 200, 'random_state': 0}
temp = array([[[[[[5.48813504e-01, 7.15189366e-01, 6.02763376e-01,
            5.44883183e-01, 4.23654799e-01, 6.45894113e-01...e-01, 7.31450249e-01,
            2.68878230e-01, 3.05965591e-01, 3.05472980e-01]]]]]],
      shape=(5, 4, 4, 7, 1, 6))
params = ['a', 'b', 'axis', 'equal_var', 'nan_policy', 'alternative', ...]
maxarg = 2
d_args = {'a': array([[[[[[5.48813504e-01, 7.15189366e-01, 6.02763376e-01,
            5.44883183e-01, 4.23654799e-01, 6.458941...       0.79329065],
          [0.33568778, 0.67971569, 0.03267317, 0.39378824, 0.0196105 ,
           0.81634554]]]]])}
intersection = set()

    @wraps(hypotest_fun_in)
    def axis_nan_policy_wrapper(*args, _no_deco=False, **kwds):
    
        if _no_deco:  # for testing, decorator does nothing
            return hypotest_fun_in(*args, **kwds)
    
        # For now, skip the decorator entirely if using array API. In the future,
        # we'll probably want to use it for `keepdims`, `axis` tuples, etc.
        if len(args) == 0:  # extract sample from `kwds` if there are no `args`
            used_kwd_samples = list(set(kwds).intersection(set(kwd_samples)))
            temp = used_kwd_samples[:1]
        else:
            temp = args[0]
    
        if not is_numpy(array_namespace(temp)):
            msg = ("Use of `nan_policy` and `keepdims` "
                   "is incompatible with non-NumPy arrays.")
            if 'nan_policy' in kwds or 'keepdims' in kwds:
                raise NotImplementedError(msg)
            return hypotest_fun_in(*args, **kwds)
    
        # We need to be flexible about whether position or keyword
        # arguments are used, but we need to make sure users don't pass
        # both for the same parameter. To complicate matters, some
        # functions accept samples with *args, and some functions already
        # accept `axis` and `nan_policy` as positional arguments.
        # The strategy is to make sure that there is no duplication
        # between `args` and `kwds`, combine the two into `kwds`, then
        # the samples, `nan_policy`, and `axis` from `kwds`, as they are
        # dealt with separately.
    
        # Check for intersection between positional and keyword args
        params = list(inspect.signature(hypotest_fun_in).parameters)
        if n_samples is None:
            # Give unique names to each positional sample argument
            # Note that *args can't be provided as a keyword argument
            params = [f"arg{i}" for i in range(len(args))] + params[1:]
    
        # raise if there are too many positional args
        maxarg = (np.inf if inspect.getfullargspec(hypotest_fun_in).varargs
                  else len(inspect.getfullargspec(hypotest_fun_in).args))
        if len(args) > maxarg:  # let the function raise the right error
            hypotest_fun_in(*args, **kwds)
    
        # raise if multiple values passed for same parameter
        d_args = dict(zip(params, args))
        intersection = set(d_args) & set(kwds)
        if intersection:  # let the function raise the right error
            hypotest_fun_in(*args, **kwds)
    
        # Consolidate other positional and keyword args into `kwds`
        kwds.update(d_args)
    
        # rename avoids UnboundLocalError
        if callable(n_samples):
            # Future refactoring idea: no need for callable n_samples.
            # Just replace `n_samples` and `kwd_samples` with a single
            # list of the names of all samples, and treat all of them
            # as `kwd_samples` are treated below.
            n_samp = n_samples(kwds)
        else:
            n_samp = n_samples or len(args)
    
        # get the number of outputs
        n_out = n_outputs  # rename to avoid UnboundLocalError
        if callable(n_out):
            n_out = n_out(kwds)
    
        # If necessary, rearrange function signature: accept other samples
        # as positional args right after the first n_samp args
        kwd_samp = [name for name in kwd_samples
                    if kwds.get(name, None) is not None]
        n_kwd_samp = len(kwd_samp)
        if not kwd_samp:
            hypotest_fun_out = hypotest_fun_in
        else:
            def hypotest_fun_out(*samples, **kwds):
                new_kwds = dict(zip(kwd_samp, samples[n_samp:]))
                kwds.update(new_kwds)
                return hypotest_fun_in(*samples[:n_samp], **kwds)
    
        # Extract the things we need here
        try:  # if something is missing
            samples = [np.atleast_1d(kwds.pop(param))
                       for param in (params[:n_samp] + kwd_samp)]
        except KeyError:  # let the function raise the right error
            # might need to revisit this if required arg is not a "sample"
            hypotest_fun_in(*args, **kwds)
        vectorized = True if 'axis' in params else False
        vectorized = vectorized and not override['vectorization']
        axis = kwds.pop('axis', default_axis)
        nan_policy = kwds.pop('nan_policy', 'propagate')
        keepdims = kwds.pop("keepdims", False)
        del args  # avoid the possibility of passing both `args` and `kwds`
    
        # convert masked arrays to regular arrays with sentinel values
        samples, sentinel = _masked_arrays_2_sentinel_arrays(samples)
    
        # standardize to always work along last axis
        reduced_axes = axis
        if axis is None:
            if samples:
                # when axis=None, take the maximum of all dimensions since
                # all the dimensions are reduced.
                n_dims = np.max([sample.ndim for sample in samples])
                reduced_axes = tuple(range(n_dims))
            samples = [np.asarray(sample.ravel()) for sample in samples]
        else:
            # don't ignore any axes when broadcasting if paired
            samples = _broadcast_arrays(samples, axis=axis if not paired else None)
            axis = np.atleast_1d(axis)
            n_axes = len(axis)
            # move all axes in `axis` to the end to be raveled
            samples = [np.moveaxis(sample, axis, range(-len(axis), 0))
                       for sample in samples]
            shapes = [sample.shape for sample in samples]
            # New shape is unchanged for all axes _not_ in `axis`
            # At the end, we append the product of the shapes of the axes
            # in `axis`. Appending -1 doesn't work for zero-size arrays!
            new_shapes = [shape[:-n_axes] + (np.prod(shape[-n_axes:]),)
                          for shape in shapes]
            samples = [sample.reshape(new_shape)
                       for sample, new_shape in zip(samples, new_shapes)]
        axis = -1  # work over the last axis
        NaN = _get_nan(*samples) if samples else np.nan
    
        # if axis is not needed, just handle nan_policy and return
        ndims = np.array([sample.ndim for sample in samples])
        if np.all(ndims <= 1):
            # Addresses nan_policy == "raise"
            if nan_policy != 'propagate' or override['nan_propagation']:
                contains_nan = [_contains_nan(sample, nan_policy)
                                for sample in samples]
            else:
                # Behave as though there are no NaNs (even if there are)
                contains_nan = [False] * len(samples)
    
            # Addresses nan_policy == "propagate"
            if any(contains_nan) and (nan_policy == 'propagate'
                                      and override['nan_propagation']):
                res = np.full(n_out, NaN)
                res = _add_reduced_axes(res, reduced_axes, keepdims)
                return tuple_to_result(*res)
    
            # Addresses nan_policy == "omit"
            too_small_msg = too_small_1d_not_omit
            if any(contains_nan) and nan_policy == 'omit':
                # consider passing in contains_nan
                samples = _remove_nans(samples, paired)
                too_small_msg = too_small_1d_omit
    
            if sentinel:
                samples = _remove_sentinel(samples, paired, sentinel)
    
            if is_too_small(samples, kwds):
                warnings.warn(too_small_msg, SmallSampleWarning, stacklevel=2)
                res = np.full(n_out, NaN)
                res = _add_reduced_axes(res, reduced_axes, keepdims)
                return tuple_to_result(*res)
    
            res = hypotest_fun_out(*samples, **kwds)
            res = result_to_tuple(res, n_out)
            res = _add_reduced_axes(res, reduced_axes, keepdims)
            return tuple_to_result(*res)
    
        # check for empty input
        empty_output = _check_empty_inputs(samples, axis)
        # only return empty output if zero sized input is too small.
        if (
            empty_output is not None
            and (is_too_small(samples, kwds) or empty_output.size == 0)
        ):
            if is_too_small(samples, kwds) and empty_output.size != 0:
                warnings.warn(too_small_nd_not_omit, SmallSampleWarning,
                              stacklevel=2)
            res = [empty_output.copy() for i in range(n_out)]
            res = _add_reduced_axes(res, reduced_axes, keepdims)
            return tuple_to_result(*res)
    
        # otherwise, concatenate all samples along axis, remembering where
        # each separate sample begins
        lengths = np.array([sample.shape[axis] for sample in samples])
        split_indices = np.cumsum(lengths)
        x = _broadcast_concatenate(samples, axis, paired=paired)
    
        # Addresses nan_policy == "raise"
        if nan_policy != 'propagate' or override['nan_propagation']:
            contains_nan = _contains_nan(x, nan_policy)
        else:
            contains_nan = False  # behave like there are no NaNs
    
        if vectorized and not contains_nan and not sentinel:
>           res = hypotest_fun_out(*samples, axis=axis, **kwds)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E           TypeError: ttest_ind() got an unexpected keyword argument 'permutations'

NaN        = np.float64(nan)
_no_deco   = False
axis       = -1
contains_nan = np.False_
d_args     = {'a': array([[[[[[5.48813504e-01, 7.15189366e-01, 6.02763376e-01,
            5.44883183e-01, 4.23654799e-01, 6.458941...       0.79329065],
          [0.33568778, 0.67971569, 0.03267317, 0.39378824, 0.0196105 ,
           0.81634554]]]]])}
default_axis = 0
empty_output = None
hypotest_fun_in = <function ttest_ind at 0x7638fc336de0>
hypotest_fun_out = <function ttest_ind at 0x7638fc336de0>
intersection = set()
is_too_small = <function _axis_nan_policy_factory.<locals>.is_too_small at 0x7638fc336ca0>
keepdims   = False
kwd_samp   = []
kwd_samples = ()
kwds       = {'permutations': 200, 'random_state': 0}
lengths    = array([7, 8])
maxarg     = 2
n_axes     = 1
n_kwd_samp = 0
n_out      = 6
n_outputs  = 6
n_samp     = 2
n_samples  = 2
nan_policy = 'propagate'
ndims      = array([6, 6])
new_shapes = [(5, 4, 4, 2, 6, np.int64(7)), (5, 4, 4, 2, 6, np.int64(8))]
override   = {'nan_propagation': True, 'vectorization': False}
paired     = False
params     = ['a', 'b', 'axis', 'equal_var', 'nan_policy', 'alternative', ...]
reduced_axes = -3
result_to_tuple = <function unpack_TtestResult at 0x7638fc3362a0>
samples    = [array([[[[[[5.48813504e-01, 4.37587211e-01, 5.68044561e-01, ...,
            1.18274426e-01, 2.64555612e-01, 6.120957...40454229, 0.97580369, 0.08510591, ..., 0.39058309,
            0.05805632, 0.81634554]]]]]], shape=(5, 4, 4, 2, 6, 8))]
sentinel   = None
shapes     = [(5, 4, 4, 2, 6, 7), (5, 4, 4, 2, 6, 8)]
split_indices = array([ 7, 15])
temp       = array([[[[[[5.48813504e-01, 7.15189366e-01, 6.02763376e-01,
            5.44883183e-01, 4.23654799e-01, 6.45894113e-01...e-01, 7.31450249e-01,
            2.68878230e-01, 3.05965591e-01, 3.05472980e-01]]]]]],
      shape=(5, 4, 4, 7, 1, 6))
tuple_to_result = <function pack_TtestResult at 0x7638fc336020>
vectorized = True
x          = array([[[[[[0.5488135 , 0.43758721, 0.56804456, ..., 0.57115297,
            0.68493794, 0.30073256],
           [0.71...63567726, 0.49569992, 0.62663675, ..., 0.39058309,
            0.05805632, 0.81634554]]]]]], shape=(5, 4, 4, 2, 6, 15))

scipy/stats/_axis_nan_policy.py:611: TypeError
=========================== short test summary info ============================
FAILED scipy/stats/tests/test_stats.py::Test_ttest_ind_common::test_ttest_many_dims[equal_var-permutations] - TypeError: ttest_ind() got an unexpected keyword argument 'permutations'
FAILED scipy/stats/tests/test_stats.py::Test_ttest_ind_common::test_ttest_many_dims[unequal_var-permutations] - TypeError: ttest_ind() got an unexpected keyword argument 'permutations'
============ 2 failed, 1190 passed, 3 skipped, 3 xfailed in 11.76s =============
```

</details>


<details>
<summary>SciPy/NumPy/Python version and system information</summary>

```shell
1.17.0.dev0+git20250621.2b2dbfc 2.3.1 sys.version_info(major=3, minor=13, micro=3, releaselevel='final', serial=0)
Build Dependencies:
  blas:
    detection method: pkgconfig
    found: true
    include directory: /usr/include/x86_64-linux-gnu/openblas-pthread/
    lib directory: /usr/lib/x86_64-linux-gnu/openblas-pthread/
    name: openblas
    openblas configuration: USE_64BITINT= DYNAMIC_ARCH=1 DYNAMIC_OLDER=1 NO_CBLAS=
      NO_LAPACK= NO_LAPACKE=1 NO_AFFINITY=1 USE_OPENMP=0 generic MAX_THREADS=64
    pc file directory: /usr/lib/x86_64-linux-gnu/pkgconfig
    version: 0.3.20
  lapack:
    detection method: pkgconfig
    found: true
    include directory: /usr/include/x86_64-linux-gnu/openblas-pthread/
    lib directory: /usr/lib/x86_64-linux-gnu/openblas-pthread/
    name: openblas
    openblas configuration: USE_64BITINT= DYNAMIC_ARCH=1 DYNAMIC_OLDER=1 NO_CBLAS=
      NO_LAPACK= NO_LAPACKE=1 NO_AFFINITY=1 USE_OPENMP=0 generic MAX_THREADS=64
    pc file directory: /usr/lib/x86_64-linux-gnu/pkgconfig
    version: 0.3.20
  pybind11:
    detection method: config-tool
    include directory: unknown
    name: pybind11
    version: 2.13.6
Compilers:
  c:
    commands: cc
    linker: ld.bfd
    name: gcc
    version: 11.4.0
  c++:
    commands: c++
    linker: ld.bfd
    name: gcc
    version: 11.4.0
  cython:
    commands: cython
    linker: cython
    name: cython
    version: 3.1.2
  fortran:
    commands: gfortran
    linker: ld.bfd
    name: gcc
    version: 11.4.0
  pythran:
    include directory: ../venv/lib/python3.13/site-packages/pythran
    version: 0.18.0
Machine Information:
  build:
    cpu: x86_64
    endian: little
    family: x86_64
    system: linux
  cross-compiled: false
  host:
    cpu: x86_64
    endian: little
    family: x86_64
    system: linux
Python Information:
  path: /home/warren/repos/git/forks/scipy-fresh/venv/bin/python3
  version: '3.13'
```

</details>

--- a/scipy/stats/tests/test_stats.py
+++ b/scipy/stats/tests/test_stats.py
@@ -5419,7 +5419,6 @@ class Test_ttest_ind_permutations:
             stats.ttest_ind([1, 2, 3], [4, 5, 6], method='migratory')
 
 
-@pytest.mark.filterwarnings("ignore:Arguments...:DeprecationWarning")
 class Test_ttest_ind_common:
     # for tests that are performed on variations of the t-test (e.g. trimmed)
     @pytest.mark.xslow()

--- a/scipy/stats/tests/test_stats.py
+++ b/scipy/stats/tests/test_stats.py
@@ -5421,12 +5421,10 @@ class Test_ttest_ind_permutations:
 
 @pytest.mark.filterwarnings("ignore:Arguments...:DeprecationWarning")
 class Test_ttest_ind_common:
-    # for tests that are performed on variations of the t-test such as
-    # permutations and trimming
+    # for tests that are performed on variations of the t-test (e.g. trimmed)
     @pytest.mark.xslow()
-    @pytest.mark.parametrize("kwds", [{'permutations': 200, 'random_state': 0},
-                                      {'trim': .2}, {}],
-                             ids=["permutations", "trim", "basic"])
+    @pytest.mark.parametrize("kwds", [{'trim': .2}, {}],
+                             ids=["trim", "basic"])
     @pytest.mark.parametrize('equal_var', [True, False],
                              ids=['equal_var', 'unequal_var'])
     def test_ttest_many_dims(self, kwds, equal_var):

